Marlin/configuration_store.cpp

```diff
@@ -64,28 +64,31 @@
  *  195  M206 XYZ  home_offset                      (float x3)
  *  207  M218 XYZ  hotend_offset                    (float x3 per additional hotend)
  *
- * Global Leveling:
+ * Global Leveling:                                 6 bytes
  *  219            z_fade_height                    (float)
+ *  223            GRID_MAX_POINTS_X                (uint8_t)
+ *  224            GRID_MAX_POINTS_Y                (uint8_t)
```

Comment:
Personally...   At least for UBL...   I think the only value of storing the GRID_MAX_POINTS_X/Y is for use during a 'Sanity Check' to make sure the world hasn't changed.   After all...   Those numbers are used by the compiler to generate the code.  You kind of want to know those are 'Correct'.  But it isn't like they are going to change.